### PR TITLE
Refactor SCSS for buttons

### LINKED ISSUE #253: Refactor SCSS
We should create reusable SCSS styles that we can use whenever we want to create a new component. To avoid duplication, style rules should be as high up in the component hierarchy as possible: for example, a style rule that applies to all modals should be applied to `Modal`, not `ChangePasswordModal`, `AddMemberModal`, etc.

Follow-up to #241

### PR DESCRIPTION
**Pull request by aidan-fitz**
Closes #253

Will do modals in a follow-up PR.

### CODE REVIEW

File: frontend/client/src/screens/AccountBranding.js [hunk 1]
```diff
@@ -178,7 +186,7 @@ const AccountBrandingBase = observer((props) => {
           <p className="section-description">
             Your dedicated account manager will gladly help you with other branding and customization needs.
           </p>
-          <div id="contact-button" onClick={onContactUsClicked}>
+          <div className="button btn-small btn-tertiary" id="contact-button" onClick={onContactUsClicked}>
```
[the remark this answers is not in the document]
Comment (aidan-fitz) [reply frontend/client/src/screens/AccountBranding.js:189]: Yes, it's supposed to. Although the account branding page isn't specified in the Figma, I kept some of the style rules for this element to preserve its width and height.

File: frontend/client/src/screens/ManageTeams.js [hunk 2]
```diff
@@ -83,10 +83,10 @@ const ManageTeamsBase = observer((props) => {
     <div className="module-container">
       <PageTitle title="Manage Members" />
       <h1>Manage Members</h1>
-      <div className="add-member-button" onClick={() => setShowAddMemberModal(true)}>
+      <button className="button btn-medium btn-tertiary add-member-button" onClick={() => setShowAddMemberModal(true)}>
```
[the remark this answers is not in the document]
Comment (aidan-fitz) [reply frontend/client/src/screens/ManageTeams.js:86]: No, we need the `align-self: flex-start` and `margin-bottom: 12px` rules

**Comment by jcolla-holla** [comment]
Added a small change to `#progress-container` in `button.scss` that resolves the jerky login experience.  I think this will resolve any similar jerky experiences across the app (I'm not sure if there even were any), but it may not.  If there are any ones that you're aware of, let's test them.  I tried to find them all and I didn't see any jerky element resizing upon the `<CircularProgress>` element loading.

Just added a simple `height` and `margin-top` to the `#progress-container` to be in line with `<button>` within any use of `<PendingButtonOperation>`.  Would have preferred something like `align-items: center` and doing away with `margin-top`s all together, but the committed approach worked and I think it will work across the board so I see no reason to change.

Preview of post-commit what the login experience looks like:

ezgif com-video-to-gif

**Comment by aidan-fitz** [comment]
Looks good. I changed `progress-container` to a class in case there is more than one on a page.

**Pull request merged**